### PR TITLE
JBPM-6681 - Support for lazy loaded variables

### PR DESCRIPTION
**Pull request by mswiderski**
depends on https://github.com/kiegroup/jbpm/pull/1095

### CODE REVIEW

**Comment by mswiderski** [comment]
Jenkins retest this

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CarInsuranceClaimCaseIntegrationTest.java [hunk 1]
```diff
@@ -71,14 +73,15 @@ public static void buildAndDeployArtifacts() {
...
+        extraClasses.put(DocumentImpl.class.getName(), DocumentImpl.class);
     }
 
     @After
     public void resetUser() throws Exception {
         changeUser(TestConfig.getUsername());
     }
 
-    @Test
+//    @Test
```
Comment (sutaakar) [inline kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CarInsuranceClaimCaseIntegrationTest.java]: Accidental leftover?

Comment (mswiderski) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CarInsuranceClaimCaseIntegrationTest.java]: yes, I knew I would do it ;)

Comment (mswiderski) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-all/src/test/java/org/kie/server/integrationtests/jbpm/cases/CarInsuranceClaimCaseIntegrationTest.java]: fixed

**Pull request merged**